bin/s3-sync.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+if [[ -z "$1" ]]; then
+    echo "Bucket name required"
+    exit 1
+fi
+
+SHORT_CACHE="--cache-control max-age=60" # 1 minute
+LONG_CACHE="--cache-control max-age=604800" # 1 week
+ARGS="--acl public-read --delete --profile mdninteractive"
+cd ../docs
+
+for path in "pages live-examples"; do
+    aws s3 sync $path s3://$1 ${SHORT_CACHE} ${ARGS}
+done
+
+for path in "css js"; do
+    aws s3 sync $path s3://$1 --exclude '*' --include 'editor-*' ${SHORT_CACHE} ${ARGS}
```

Comment:
include and exclude syntax based on http://docs.aws.amazon.com/cli/latest/userguide/using-s3-commands.html